src/main/java/com/epam/tc/gdi/hw1/site/pages/MetalsColorsPage.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.tc.gdi.hw1.site.pages;
+
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.tc.gdi.hw1.site.pages.components.complex.Header;
+import com.epam.tc.gdi.hw1.site.pages.components.complex.MetalsColorsForm;
+
+public class MetalsColorsPage extends WebPage {
+
+    @Css("form[class=form]")
+    public MetalsColorsForm metalsColorsForm;
+
+    @Css("ul[class=\"panel-body-list results\"]>li")
+    public WebList result;
+
+    public void shouldBeOpened() {
+        if (!getTitle().equalsIgnoreCase("metals and colors")) {
```

Comment:
The better way is to create annotation Title and use built-in method to check title

Reply:
Done